python/samples/04.ai.a.teamsChefBot/README.md

```diff
@@ -206,7 +210,7 @@ You can also use the Teams Toolkit CLI to run this sample.
     - Use the current `https` URL you were given by running ngrok. Append with the path `/api/messages` used by this sample.
     - Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
 
-1. Update the `.env` configuration for the bot to use the Microsoft App Id and App Password from the Bot Framework registration. (Note the App Password is referred to as the "client secret" in the Azure Portal and you can always create a new client secret anytime.)
+1. Update the `.env` configuration for the bot to use the Microsoft App Id and App Password from the Bot Framework registration, as well as the Azure Open AI key, endpoint and `text-davinci-003` model's deployment name. (Note the App Password is referred to as the "client secret" in the Azure Portal and you can always create a new client secret anytime.)
```

Comment:
Do we want to use a chat completion model instead of text completion? (since text completion is going to be deprecated in January)

Reply:
Good catch! Will create another task to track this. This PR aims to fix readme so people can follow it to try the samples via various ways.